roofit/roofitcore/inc/RooAbsReal.h

```diff
@@ -108,7 +108,7 @@ class RooAbsReal : public RooAbsArg {
 
   virtual Double_t getValV(const RooArgSet* normalisationSet = nullptr) const ;
 
-  virtual RooSpan<const double> getValBatch(std::size_t begin, std::size_t maxSize, const RooArgSet* normSet = nullptr) const final;
+  virtual RooSpan<const double> getValBatch(std::size_t begin, std::size_t maxSize, const RooArgSet* normSet = nullptr) = delete;
```

Comment:
A comment what happened to the function and how people who were using it can update is needed (if there are any :-) ).